python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -756,13 +764,16 @@ cdef class Column:
         return Column.from_libcudf(move(c_result))
 
     @staticmethod
-    cdef Column _from_gpumemoryview(
+    cdef Column _wrap_nested_list_column(
```

Comment:
Note: I changed the name of this utility because it can now wrap the resulting base column in a nested list column.